apps/src/templates/sectionProgress/summary/VirtualizedSummaryView.jsx

```diff
@@ -158,6 +160,7 @@ class VirtualizedSummaryView extends Component {
           cellRenderer={this.cellRenderer}
           columnWidth={this.getColumnWidth}
           columnCount={columnCount}
+          ref="summaryView"
```

Comment:
interesting, i wasn't familiar with refs! however, according to the [react docs](https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs), string refs are deprecated. 
```
We advise against it because string refs have some issues, are considered legacy, and are likely to be removed in one of the future releases. 
```
would one of the other methods of creating a ref listed on that page work here?

Reply:
Thanks for pointing that out! Updated to use the callback method.